core/callback.go

```diff
@@ -19,7 +18,7 @@ type RestGenCallback interface {
 	// This is returned from the status endpoint
 	Config() validator.Validator
 	// HandleError allows custom HTTP errors to be added to `sys-go` errors
-	HandleError(ctx context.Context, w http.ResponseWriter, kind common.Kind, message string, cause error)
+	MapError(ctx context.Context, cause error) *common.HTTPError
```

Comment:
Minor request. Can we rename `cause` to `err`. The actual error passed is a wrapped error, not the original cause.